src/sonardevices/OculusSonarImage.cpp

```diff
@@ -0,0 +1,3 @@
+
+#include <sonar_driver/sonardevices/OculusSonarImage.h>
```

Comment:
@EulersBoiler If this file does not add any implementation, I think it can be safely removed.